### PR TITLE
aTask1c, Added validation to application entity, Removed useless method from ap…

### PR DESCRIPTION
**Pull request by turnerror**
…plicant model and changed saveapplicant controller to apply changes.

### CODE REVIEW

File: src/Controllers/SaveApplicantController.php [hunk 1]
```diff
@@ -31,40 +34,46 @@ public function __construct(ApplicantModel $applicantModel)
      * @param array $args The arguments array
      *
      * @return Response, will return the data from successfulRegister and the statusCode, via Json.
+     * @throws Exception
      */
-    public function __invoke(Request $request, Response $response, Array $args)
+    public function __invoke(Request $request, Response $response, array $args)
```
Comment (amandahrdmn) [inline src/Controllers/SaveApplicantController.php:39]: why change the Array to array?

File: src/Validators/PhoneNumberValidator.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    /**
+     * Make sure the phone Number is valid
+     *
+     * @param string $phoneNumber
+     * @return string|null
+     * @throws Exception
+     */
+    public static function validatePhoneNumber(string $phoneNumber)
+    {
+        if ($phoneNumber == '') {
```
Comment (amandahrdmn) [inline src/Validators/PhoneNumberValidator.php:22]: don't you want to throw the 400 error if you don't have a phone number?

File: tests/Validators/PhoneNumberValidatorTest.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+namespace Tests\Validators;
+
+use Portal\Validators\PhoneNumberValidator;
+
+class PhoneNumberValidatorTest extends \PHPUnit\Framework\TestCase
+{
+
+    public function testValidatePhoneNumberSuccess()
+    {
+        $numbers = ['786780670434', '+31 166 094'];
```
Comment (amandahrdmn) [inline tests/Validators/PhoneNumberValidatorTest.php:12]: need to check what happens if you don't input a phone number

**Comment by amandahrdmn** [comment]
just realised that StringValidatorTest also needs checks for empty strings in the validateLength function

File: src/Controllers/SaveApplicantController.php (hunk 1 above)
Comment (turnerror) [reply src/Controllers/SaveApplicantController.php:39]: psr standard

**Comment by turnerror** [comment]
> just realised that StringValidatorTest also needs checks for empty strings in the validateLength function

The validate length function should be able to be empty. That's why there's 2, validateLength and validateExistsAndLength

**Pull request merged**